ext/node/polyfills/worker_threads.ts

```diff
@@ -424,10 +424,21 @@ export function receiveMessageOnPort(port: MessagePort): object | undefined {
   return { message: deserializeJsMessageData(data)[0] };
 }
 
+class NodeMessageChannel {
+  port1: MessagePort;
+  port2: MessagePort;
+
+  constructor() {
+    const { port1, port2 } = new MessageChannel(true);
```

Comment:
Not happy that nodeWorker flag needs to be public for MessageChannel but I'm not aware of a better choice here. Is there an alternative?